src/common/util.ts

```diff
@@ -1,12 +1,30 @@
 import { isAfter, parseISO, isBefore, format } from 'date-fns';
-import { Parameter } from './types';
+import { Parameter, TranslationObject } from './types';
 
-// eslint-disable-next-line import/prefer-default-export
 export const isActive = (startDate: string, endDate: string): boolean => {
   const now = new Date();
   return isAfter(now, parseISO(startDate)) && isBefore(now, parseISO(endDate));
 };
 
+const isPast = (endDate: string): boolean => {
+  const now = new Date();
+  return isAfter(now, parseISO(endDate));
+};
+
+export const uiState = (
```

Comment:
This could use a bit more descriptive name.

Reply:
true, renamed